src/gpu/intel/jit/jit_reduction_injector.cpp

```diff
@@ -41,6 +41,11 @@ void jit_reduction_injector_f32<hw>::sum_fwd(
     eadd(h, simd, acc, acc, val);
 }
 template <gpu_gen_t hw>
+void jit_reduction_injector_f32<hw>::amax_fwd(
+        int simd, const ngen::GRF &acc, const ngen::GRF &val) {
+    h.amax_(simd, acc, acc, val);
```

Comment:
Instead of defining an amax pseudo-instruction, just write here what you'd like to do:
```suggestion
    h.max_(simd, acc, acc, abs(val));
```

Reply:
I didn't know that it is fine to pass a variable of type ngen::GRF to abs(). Then, this will reuses the h.max_ function. Thanks.

Reply:
Just a note: the jit_reduction_injector is currently only used in one dev-mode-only (i.e. `-DDNNL_DEV_MODE=on`) reduction implementation. If you want to test that this works, you'll need to compile in dev mode and see `src/gpu/intel/jit/jit_reduction.cpp` for the limited set of allowed cases for this implementation. Otherwise, the library currently uses opencl-based kernels exclusively.